data/tr2/ship/cfg/TR2X_strings_gm.json5

```diff
@@ -0,0 +1,474 @@
+{
+    // For usage, refer to the documentation here:
+    // https://github.com/LostArtefacts/TRX/blob/stable/docs/GAME_STRINGS.md
+
+    "levels": [
+        // 0. Lara's Home
+        {
+            "title": "Lara's Home",
+        },
+
+        // 1. The Cold War
+        {
+            "title": "The Cold War",
+            "objects": {
+                "tiger": {"name": "Snow Leopard"},
+                "key_1": {"name": "Guardroom Key"},
+                "key_2": {"name": "Shaft 'B' Key"},
+            },
+        },
+
+        // 2. Fool's Gold
+        {
+            "title": "Fool's Gold",
+            "objects": {
+                "key_1": {"name": "CardKey 1"},
+                "key_4": {"name": "CardKey 2"},
+                "puzzle_1": {"name": "Circuit Board"},
+            },
+        },
+
+        // 3. Furnace of the Gods
+        {
+            "title": "Furnace of the Gods",
+            "objects": {
+                "big_spider": {"name": "Polar Bear"},
+                "spider": {"name": "Wolf"},
+                "puzzle_1": {"name": "Mask Of Tornarsuk"},
+                "puzzle_2": {"name": "Gold Nugget"},
+            },
+        },
+
+        // 4. Kingdom
+        {
+            "title": "Kingdom",
+            "objects": {
+                "tiger": {"name": "Snow Leopard"},
+                "puzzle_1": {"name": "Mask Of Tornarsuk"},
+            },
+        },
+
+        // 5. Nightmare in Vegas
+        {
+            "title": "Nightmare In Vegas",
+            "objects": {
+                "key_1": {"name": "Hotel Key"},
+                "puzzle_1": {"name": "Elevator Junction"},
+                "puzzle_2": {"name": "Door Circuit"},
+            },
+        },
+    ],
+
+    "demos": [
+    ],
+
+    "cutscenes": [
+    ],
+
+    "objects": {
+        "small_medipack":        {"name": "Small Medipack"},
```

Comment:
No need to provide all the defaults- they're out there in the exe and in the main GF file as means of showing what can be changed.